app-emulation/wine-vanilla/files/wine-vanilla-6.12-winegcc-equals-args.patch

```diff
@@ -0,0 +1,13 @@
+https://bugs.winehq.org/show_bug.cgi?id=51413
+https://bugs.gentoo.org/800809
```

Comment:
We need to keep in sync with ::wine, so please make sure this change makes it to both repos